addons/test_website/views/test_model_views.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0"?>
+<odoo>
+
+<!-- test.model views -->
+<record id="test_model_view_kanban" model="ir.ui.view">
+    <field name="name">test.model.kanban</field>
+    <field name="model">test.model</field>
+    <field name="arch" type="xml">
+        <kanban js_class="website_pages_kanban" class="o_kanban_mobile" action="open_website_url" type="object" sample="1">
+            <field name="name"/>
+            <field name="website_url"/>
+            <templates>
+                <t t-name="kanban-box">
+                    <div class="oe_kanban_global_click d-flex flex-column">
+                        <div class="row mb-auto">
+                            <strong class="col-8">
+                                <span class="o_text_overflow" t-esc="record.name.value"/>
+                                <div class="text-muted" t-if="record.website_id.value" groups="website.group_multi_website">
+                                    <i class="fa fa-globe me-1" title="Website"/>
+                                    <field name="website_id"/>
+                                </div>
+                            </strong>
+                        </div>
+                        <div class="border-top mt-2 pt-2">
+                            <field name="is_published" widget="boolean_toggle"/>
+                            <t t-if="record.is_published.raw_value">Published</t>
+                            <t t-else="">Not Published</t>
+                        </div>
+                    </div>
+                </t>
+            </templates>
+        </kanban>
+    </field>
+</record>
+<record id="test_model_view_list" model="ir.ui.view">
+    <field name="name">Test Model Pages Tree</field>
+    <field name="model">test.model</field>
+    <field name="priority">99</field>
+    <field name="arch" type="xml">
+        <tree js_class="website_pages_list" type="object" action="open_website_url" multi_edit="1">
+            <field name="name"/>
+            <field name="website_url"/>
+            <field name="website_id" groups="website.group_multi_website"/>
+        </tree>
+    </field>
+</record>
+<record id="action_test_model" model="ir.actions.act_window">
+    <field name="name">Test Model Pages</field>
+    <field name="res_model">test.model</field>
+    <field name="view_mode">tree,kanban,form</field>
+    <field name="view_ids" eval="[(5, 0, 0),
+        (0, 0, {'view_mode': 'tree', 'view_id': ref('test_model_view_list')}),
+        (0, 0, {'view_mode': 'kanban', 'view_id': ref('test_model_view_kanban')}),
+    ]"/>
+</record>
+<menuitem id="menu_test_model_pages"
+    parent="website.menu_content"
+    sequence="20"
+    name="Test Models"
+    action="action_test_model"/>
```

Comment:
Hm.. This is annoying on runbot, because those will be shown.
I guess I could either:
- show those only in debug (but would need the test to be run in debug)
- not create those in XML but just in the python before the test
- directly land on the form view

The third one is surely the way to go, I'll do that tomorrow, but it's just about a detail in the test so the fix can probably be reviewed in the meantime.

Reply:
Agreed 3 > 2 > 1

Reply:
Changed: https://github.com/odoo/odoo/compare/f145e55ab0db47c4249286ac222ddcda289ef7a3..e904767890dab8682acc0040f42d36cbd8ae6712